subsys/bluetooth/host/conn.c

```diff
@@ -1776,6 +1787,16 @@ int bt_conn_disconnect(struct bt_conn *conn, u8_t reason)
 			bt_le_scan_update(false);
 		}
 		return 0;
+	case BT_CONN_CONNECT_DIR_ADV:
```

Comment:
Directed Advertising depends on Peripheral Role, all code here needs to be compiled out, not just the bt_le_adv_stop().

Reply:
It is handled in a similar way as the ```BT_CONN_CONNECT_SCAN``` state. Should I align both cases?